.chloggen/k8sobjects-data-structure.yaml

```
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: k8sobjectsreceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: use same data structure for objects received via `pull` and `watch` mode
```

It is preferred change-log entries to be in the form of user facing messages about what is fixed, added etc without surfacing the technical details.

How about sth the following:

```suggestion
note: Ensure `pull` and `watch` modes provide same set of attributes and resource attributes.
```